tasklist/src/pages/listNotification/listNotification.ts

```diff
@@ -0,0 +1,69 @@
+import { Component } from '@angular/core';
+
+import { ModalController, NavController, NavParams, ToastController } from 'ionic-angular';
+
+import { NotificationService } from "../../services/notificationService";
+
+@Component({
+  selector: 'list-notification',
+  templateUrl: 'listNotification.html',
+  providers: [NotificationService]
+})
+export class ListNotification {
+  notifications: Array<any>;
+
+  constructor(
+    public navCtrl: NavController,
+    public navParams: NavParams,
+    public toastCtrl: ToastController,
+    private notificationService: NotificationService
+  ) {
+    this.getDataFromApi();
+  }
+
+  getDataFromApi() {
+    this.notificationService.listNotifications().subscribe(
+      response => {
+        this.notifications = response.json()
+      },
+      err => {
+        let toast = this.toastCtrl.create({
+          message: 'Error : Connection server',
+          dismissOnPageChange: true
+        });
+        toast.present();
+      }
+    );
+  }
+
+  notificationTapped(event, notification) {
+    // Mark notification as read or unread
+    let handler;
+    if( notification.is_read ) {
+      handler = this.notificationService.markNotificationAsUnread(notification.id);
```

Comment:
I don't understand (but i've no tried to run the code for the moment so it's maybe my bad).

This line say (and the line 51 too) we can put a notification already `read` as `unread` ? Beacause it's seems strange to me to allow this action.